### PR TITLE
Added retries in executing alter queries on Target YugabyteDB for error "Errors occurred while reaching out to the tablet servers"

### PR DESCRIPTION
**Pull request by priyanshi-yb**
On a single node YB cluster executing multiple alter statements after the data load is erroring out with error - `Errors occurred while reaching out to the tablet servers`. Added 5 retries with constant backoff of 10 sec for that.
Fix - https://yugabyte.atlassian.net/browse/DB-9869
and few misc UI changes
Jenkins - https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline/1864

### CODE REVIEW

File: yb-voyager/src/tgtdb/yugabytedb.go [hunk 1]
```diff
@@ -1055,24 +1056,34 @@ func (yb *TargetYugabyteDB) alterColumns(tableColumnsMap map[string][]string, al
...
+					return false, fmt.Errorf("closing batch of queries to alter columns for table(%s): %w", qualifiedTableName, err)
+				}
+				return false, nil
+			})
+			if err != nil {
+				log.Errorf("error in altering columns for table(%s): %v", qualifiedTableName, err)
+				if !strings.Contains(err.Error(), "while reaching out to the tablet servers") {
+					return err
+				}
+				sleepIntervalSec += 10
```
Comment (amit-yb) [inline yb-voyager/src/tgtdb/yugabytedb.go:1081]: Keep the sleep constant `10` seconds (do not increase the backoff interval).

File: yb-voyager/src/tgtdb/yugabytedb.go [hunk 2]
```diff
@@ -1055,24 +1056,34 @@ func (yb *TargetYugabyteDB) alterColumns(tableColumnsMap map[string][]string, al
...
+				log.Infof("retrying after %d seconds for table(%s)", sleepIntervalSec, qualifiedTableName)
+				time.Sleep(time.Duration(sleepIntervalSec) * time.Second)
+			} else {
+				break
 			}
```
Comment (amit-yb) [inline yb-voyager/src/tgtdb/yugabytedb.go:1086]: Instead, you should use code structure like:
```
for ... {
 ...
  if err != nil {
    ...
    continue // next retry
  }
  break
}
```

**Review by amit-yb — APPROVED** [review]
LGTM with minor comments.

File: yb-voyager/src/tgtdb/yugabytedb.go [hunk 3]
```diff
@@ -1055,24 +1056,33 @@ func (yb *TargetYugabyteDB) alterColumns(tableColumnsMap map[string][]string, al
...
+				}
+				return false, nil
+			})
+			if err != nil {
+				log.Errorf("error in altering columns for table(%s): %v", qualifiedTableName, err)
+				if !strings.Contains(err.Error(), "while reaching out to the tablet servers") {
+					return err
+				}
+				log.Infof("retrying after %d seconds for table(%s)", sleepIntervalSec, qualifiedTableName)
+				time.Sleep(time.Duration(sleepIntervalSec) * time.Second)
```
Comment (sanyamsinghal) [inline yb-voyager/src/tgtdb/yugabytedb.go:1082]: it will be fixed constant `10 sec` sleep after every retry, right? Where is backoff

**Review by sanyamsinghal — APPROVED** [review]

**Pull request merged**